.github/workflows/nodejs.yml

```diff
@@ -27,6 +27,8 @@ jobs:
       - name: Update npm to 8.x
         run: |
           npm i -g npm@8
+          npm --version
+          node --version
```

Comment:
do we need this here or is it just debug?

Reply:
debug, will remove